<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="81" started-at="2018-02-07T06:18:45Z" finished-at="2018-02-07T06:18:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="81" started-at="2018-02-07T06:18:45Z" finished-at="2018-02-07T06:18:45Z">
      <class name="com.example.test.FormTest">
        <test-method status="FAIL" signature="dpGet(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.example.test.FormTest@5df0733a]" name="dpGet" duration-ms="0" started-at="2018-02-07T14:18:45Z" data-provider="feeser" finished-at="2018-02-07T14:18:45Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void com.example.test.FormTest.dpGet(java.lang.String,java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : feeser]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void com.example.test.FormTest.dpGet(java.lang.String,java.lang.String,java.lang.String) throws java.lang.Exception requires a @DataProvider named : feeser
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:268)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:417)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dpGet -->
      </class> <!-- com.example.test.FormTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
